src/main/java/binnie/core/genetics/BreedingSystem.java

```diff
@@ -145,12 +144,15 @@ public void calculateArrays() {
 				allBranches.add(branch);
 			}
 		}
+
 		if (getSpeciesRoot().getMutations(false) != null) {
 			Set<IMutation> mutations = new LinkedHashSet<>();
 			mutations.addAll(getSpeciesRoot().getMutations(false));
 			if (this == Binnie.Genetics.beeBreedingSystem) {
-				mutations.addAll(ExtraBeeMutation.mutations);
+				// TODO !!! check for needed !!!
```

Comment:
I looked into [the original project on bitbucket](https://bitbucket.org/binnie567/binnie-mods/commits/d286440deae0c96a352ec239b28e1331bd35108d#Lcore-src/binnie/core/genetics/BreedingSystem.javaF158) where i have read access to the repository, but it is not publicly visible.